common/src/claims.rs

```diff
@@ -85,6 +85,9 @@ pub enum Scope {
     /// of DNS-01 challenge completion.
     GatewayCertificateRenew,
 
+    /// Perform Pro tier action
+    Pro,
+
```

Comment:
Hmm, I'm not sure if this should be a scope. Shouldn't we rather have scopes for actions a pro user is allowed to take, that are only assigned to users with the pro account_tier? E.g. create up to 15 projects, create custom domains and provision RDS (when those are implemented).

Reply:
Also, are we creating a breaking change here, or is adding new fields fine? :thinking:

Reply:
I tried going down that road, but all our claims logic is based on checking if a Vec contains a scope, which means we can't parameterize this enum easily (i.e. `ProjectCreate(u32)`). The admin scope is done in the same way. New field is not breaking.

Reply:
As we discovered, breaking changes here are not preferred :hot_face: 
I'm all for reworking claims to be a struct with bools and options, but that would be a huge breaking change for another time.

Reply:
Could there be something like an ExtraProjects variant, only assigned to users with the pro tier? And then if anyone doesn't have that scope, they get 3 projects.

Reply:
This does not scale to the optimal case, where the limit is fully adjustable for any group of users or individuals.
> struct with bools and options

This would allow things of that sort.